docs/blog/newsletter-april-2020.rst

```diff
@@ -0,0 +1,98 @@
+
+------------------------------------------
+Pros and cons of the docs-as-code approach
+------------------------------------------
+
+The `docs as code <https://www.writethedocs.org/guide/docs-as-code/>`_ approach to documentation means using the same tools and workflows for the documentation, as developers use for the code. Writers using this approach are often integrated into the product team, too - but does that mean your docs should be in the code repository too? Here are the community's thoughts:
+
+In favour of *keeping* docs and code in the same repo:
+
+- Less friction to docs contributions: developers don't have to leave their projects to work on the docs.
+- Comprehensive pull requests: a single pull request can include code changes and the associated docs updates.
+- Eliminates the manual step of coordinating releases and pull request merges between repos.
+- Facilitates contributions from the entire product team, which can mean more perspectives are reflected in the documentation.
+
+Pros of *separating* docs and code:
+
+- Less noise in the code repo. For example, docs commits won't appear in automated change logs and team notifications.
+- Mistakes in the docs repo cannot affect the code.
+- A separate docs repository can be for new team members to understand.
+- Build time will increase as code + docs size increases (although parallelization can mitigate this problem). If docs are in a separate repo, writers don't have to wait for code builds (and devs don't have to wait for docs builds).
+- It takes technical knowledge to set up, customize, and maintain a combined repo and toolchain - so you need either that knowledge yourself, or development resources.
+- Some docs publication tools don't play well with a code development workflow (for example, FrameMaker, InDesign, or Word).
+
+--------------------------
+Salary Survey Speculations
+--------------------------
+
+A chorus of thank-yous greeted the publication of the `WTD salary survey for 2019 </surveys/salary-survey/2019/>`_ -- and were quickly followed by numerous anecdotes, speculation about The State of the Profession (technical writing, that is), and more. Highlights of the conversation:
+
+* It was hard to compare absolute salary information across countries and regions without information about cost of living. Some folks offered their sense of differences, though -- for example, lower salaries in Pennsylvania than in New York were thought to be more than offset by the lower cost of living.
+* Some people thought that salaries in expensive cities seemed low; others seemed to realize just how low their own salaries were compared with the average for their locale.
+* Some felt that the data gave them what they needed to negotiate raises or, more generally, better compensation packages! Just what we like to hear.
```

Comment:
> Just what we like to hear.

this bit of editorial snuck in. Should we mark it somehow

Reply:
You're right, that crosses the line a bit. I'll remove it.